docs/UserGuide.md

```diff
@@ -214,11 +271,7 @@ Format: `deleteHouse [n/NAME] [street/STREET]  [blk/BLOCK] [level/LEVEL] [unitNo
 **Note:** A seller with one house can have his house deleted. The seller would have no houses to his name then but would still exist.
 </box>
 
-Examples:
-* `deleteHouse n/John Doe type/HDB street/Clementi Ave 2 blk/311 level/02 unitNo/25 postal/578578 price/10000`
-* `deleteHouse n/John Doe type/Condominium street/Clementi Ave 2 blk/N/A level/02 unitNo/25 postal/578578 price/10000`
-* `deleteHouse n/John Doe type/Condominium street/Clementi Ave 2 level/02 unitNo/25 postal/578578 price/10000`
-* `deleteHouse n/John Doe type/Landed street/Clementi Ave 2 unitNo/25 postal/578578 price/10000`
+Examples: Largely similar to addHouse commands, except the addHouse command word is now deleteHouse.
```

Comment:
I think it might be better to still give one example at least for this one, but yeah not more than one since it's similar.